docs/dev/api/insights.md

```diff
@@ -2874,6 +2881,12 @@ Return an array of test cases (grouped by name) with statistical details.
      <td><p><small>| QUERY | OPTIONAL | ARRAY OF STRINGS |</small></p><p>Limit results to only those run on the specified framework.</p></td>
     </tr>
   </tbody>
+  <tbody>
+    <tr>
+     <td><code>device_group</code></td>
+     <td><p><small>| QUERY | OPTIONAL | STRING |</small></p><p>If <code>source</code> is <code>rdc</code>, return results only for the specified device group. Available values are: <ul><li><code>private</code></li><li><code>public</code></li></ul></p></td>
+    </tr>
```

Comment:
Is it worth to indicate that `device_group` only applies to RDC tests?

Reply:
I did, isn't it enough? "If <code>source</code> is <code>rdc</code>, return results only for the specified device group. "

Reply:
Ah, of course it's enough.